src/fraud.rs

```diff
@@ -0,0 +1,38 @@
+use jsonrpsee::proc_macros::rpc;
+use jsonrpsee::core::{Error};
+use protobuf::Message;
+use crate::proto::{byzantine};
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Serialize, Deserialize)]
+#[serde(rename_all = "lowercase")]
+pub enum ProofType {
+    BadEncoding
+}
+
+#[derive(Deserialize, Debug)]
+pub struct ProofResponse {
+    pub proof_type: String,
+    pub data: Vec<u8>
```

Comment:
```suggestion
    pub data: Vec<u8>,
```

see the naming conventions in a struct for example [here](https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter)